website/docs/sql/datatype/array.md

```diff
@@ -0,0 +1,95 @@
+# Array Type
+
+An array is a sequential collection of elements packed into a single SQL value.
+Arrays are useful to model dependent information with an inherent ordering in a single row – for example, time-series data from longitudinal studies, or embedding vectors from machine learning models. 
+
+Generally, array values are denoted in curly braces with elements separated by commas, like so:
+
+```
+{"Lorem", "ipsum", "dolor", "sit", "amed"}
+```
+
+
```

Comment:
Either use two or one space before captions. Align with rest of documentation.

Reply:
If this is important we should lint it. [2749843](https://github.com/tableau/hyper-db/pull/110/commits/27498430ec1738af4c4028b7ac560f1c70f06fb4)